packages/jest-phabricator-coverage/src/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ * @flow
+ */
+
+'use strict';
+
+import type {
+  AggregatedResult,
+  CoverageMap,
+  TestResult
+} from 'types/TestResult';
+
+type PhabricatorReport = {|
+  phabricatorReport: Array<Object>
```

Comment:
Guess you're refering to the `Object`, we are building an object that has a lot of loose properties so flow was complaining a lot, also I had to change this to not be an exact type as it will be merged to the result object later.
I'm going to change the implementation anyway to use `jest-util/formatTestResults` so I'll fix it then.